editor/xml_feature.cpp

```diff
@@ -532,7 +533,10 @@ XMLFeature ToXML(osm::EditableMapObject const & object, bool serializeType)
 
   object.ForEachMetadataItem([&toFeature](string_view tag, string_view value)
   {
-    toFeature.SetTagValue(tag, value);
+    if (osm::isSocialContactTag(tag) && value.find('/') != std::string::npos)
+      toFeature.SetTagValue(tag, osm::socialContactToURL(tag, value));
```

Comment:
Would it be easier to avoid any logic here, and store it as-is? But apply the stripping logic (in C++ core) when a user presses "Save" in the editor window?

Reply:
We do strip full URL in Editor. We can't send to OSM partial URLs like 'contact:instagram=p/BvkgKZNDbqN'. [Wiki](https://wiki.openstreetmap.org/wiki/Key:contact:*) says that either username of full URL is expected. This part of code converts all shortened URLs to full URLs before sending to OSM API.